data/Santini_2016/metadata.yml

```diff
@@ -0,0 +1,349 @@
+    the mass of the displaced hydrated branch. The same branches were oven dried for
+    5 days at 60 °C to attain constant weight. Density of intact branches was calculated
+    by dividing dry mass by hydrated branch volume. Wood density was measured following
+    the same methods as for measuring density of intact branches but after carefully
+    debarking our intact branches with a razor blade.
+- var_in: wood density total, including bark and pith (g/cm3)
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 6
+  method_context: total wood
+  methods: Material to measure density of intact branches and wood density was stored
+    in paper bags and was maintained cool in an insulated container until laboratory
+    analysis. Density of intact branches was measured from ~5 cm length branch segments.
+    Branches were placed in water for 2 days to obtain full branch hydration. We measured
+    the mass of the displaced hydrated branch. The same branches were oven dried for
+    5 days at 60 °C to attain constant weight. Density of intact branches was calculated
+    by dividing dry mass by hydrated branch volume. Wood density was measured following
+    the same methods as for measuring density of intact branches but after carefully
+- var_in: stem Vessel wall thickness (um)
+  unit_in: um
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
+  methods: Branches were collected and stored in sealed vials containing 50% ethanol.
+    Cross sections of each branch were fixed in formalin-acetic acid-alcohol (FAA)
+    for 7 days and then placed in 70 % ethanol for 2 days; we repeated this washing
+    process twice. Micro-sections of all species with the exception of E. vespertilio
+    were prepared with a sledge Leica SM2010R microtome. Micro-sections of E. vespertilio
+    were pre-treated with ethanol (8 h), xylene (4 h) and finally embedded in paraffin
+    with a Shandon Histocentre 3 histology-embedding centre (Thermo Fisher Scientific,
+    Australia) before cutting with a Microm HM325 rotatory microtome (Thermo Fisher
+    Scientific, Australia). Micro-sections of all species were double stained with
+    safranin-alcian blue. We photographed the micro-sections with a Leica DM750 microscope
+    and used the software Image J 1.48v (National Institutes of Health, USA) to measure
+    xylem traits. We determined the minor (a) and major (b) axis diameters of xylem
+    vessels to calculate vessel area (VA) following Eq. (1). We also determined vessel
+    density (VD) (Eq. 2) to calculate conductive area (CA) of each species as in Eq.
+    (3) (Lewis 1992)
+- var_in: d13C
+  unit_in: '{delta}[ppth]'
+  trait_name: leaf_delta13C
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
```

Comment:
Wondering where 9 is from, sorry if already addressed! (And also stem_vessel_lumen_fraction)

Reply:
Yes, replaced with actual # of individuals